### ISSUE TITLE
"Follows you" sometimes is not shown ["follows you" badge for remote profile popup]

### ISSUE THREAD

**Issue by moshpirit** [hannesmannerheim/qvitter#178]
I get the notification: "someone has follow you" so I click on the username of this "someone", the profile resume is shown but not the "follows you" message. Then I think: OK, maybe (s)he stopped following me, but then, I check out the people that (s)he's following and I'm there.

It recently happened with loadaverage.org, social.mxchange.org, micro.fragdev.com, gnusocial.net and indy.im. With people of my own server works perfectly as I've seen.

Checked on quitter.es and status.vinilox.eu

**Comment by hannesmannerheim** [comment]
not all instances display the so called "activity notices", e.g. quitter.se. i don't know about the others. in that case it's just like you describe, it's only visible in your notifications and in their followers. 

but i notices that twitter recently started displaying recent followings on peoples profiles, so maybe we should do that in qvitter aswell, i'm not sure what i think.

**Comment by moshpirit** [comment]
I think I didn't describe it well. What I meant is: 

A follows B
B get a notification and check A's profile 
Then, B see in the A's resume (name, bio, activity and last note) a message:
"follows you" 

This message is the one that I could a few weeks ago, and not now

**Comment by hannesmannerheim** [comment]
i understand. maybe the admin disabled activity messages, or there's been some change in gnusocial removing them. or there is a bug that's causing them not to be created or shown. 

i don't think it has to do with qvitter.

**Comment by hannesmannerheim** [comment]
hm, but there is one outputted in your profile 
![skarmavbild 2015-05-22 kl 12 51 32](https://cloud.githubusercontent.com/assets/2437812/7769061/5eab40a2-0081-11e5-9e48-ff1cb9ba136a.png)

**Comment by hannesmannerheim** [comment]
and this user produced an activity notice when following you 2 days ago:
![skarmavbild 2015-05-22 kl 12 52 57](https://cloud.githubusercontent.com/assets/2437812/7769085/8e09479a-0081-11e5-908e-fef5c5243aef.png)

**Comment by moshpirit** [comment]
No, it's not this kind of activity, I mean the resume of an account, what I see when I click on another profile: photo and background, queets, following, groups and the last one queet. 

I'm not in home, I'll post later a picture.

**Comment by hannesmannerheim** [comment]
my screenshots _are_ from the profile! from quitter.es/moshpirit and quitter.es/allanviega

**Comment by hannesmannerheim** [comment]
oh, now i understand.

**Comment by hannesmannerheim** [comment]
the "profile popup"/resume/summary is only for remote users, not local. and whether or not the "started following X" activity notice shows as the last notice depends on the remote instance. if the remote instance has activity notices turned on and the activity is the latest notice.

**Comment by hannesmannerheim** [comment]
example from quitter.es displaying an indy.im user
![skarmavbild 2015-05-22 kl 13 15 43](https://cloud.githubusercontent.com/assets/2437812/7769517/b65e9490-0084-11e5-822f-affa82dc3748.png)

**Comment by moshpirit** [comment]
Exactly :D excuse me if I didn't explain myself correctly, maybe I should post some photo before I went out

**Comment by hannesmannerheim** [comment]
ohh! NOW i think i understand! sorry. the small "follows you" badge beside the username  is missing from the profile popup.

that is correct, but it has actually never existed for remote users in the profile popup. i have that as a feature on the todo list.

**Comment by hannesmannerheim** [comment]
so, you did most certainly not have that a few weeks ago.

**Comment by moshpirit** [comment]
Mm now I see... I didn't know it wasn't implemented. Ok thank you very much!!

**Comment by hannesmannerheim** [comment]
sorry for being stupid and slow to understand!

**Comment by moshpirit** [comment]
Haha, yes, I thought I saw it before in other instances popup, in fact, this pop up only is shown for other instances, isn't it?

**Comment by moshpirit** [comment]
Don't worry, next time I'll post a screenshot :-) thanks for your work!

**Comment by hannesmannerheim** [comment]
yep, it's only for remote instances. but i think i will make it for all users, even local users. now you don't know if clicking a user will open a popup or switch page, which is not good.

**Comment by moshpirit** [comment]
Perfect :-) I think I remember I asked you for this change

**Comment by moshpirit** [comment]
**BUG REPORT**

First, thanks for the works! the feature is really cool! but I've found a bug, when I click on her username *from notifications*, it doesn't show the popup of her profile, but mine.

![zzzzzzzzzzz](https://cloud.githubusercontent.com/assets/7888062/8022035/b0d8f16e-0cb9-11e5-9431-ae91a986b5ca.png)

I've just checked it [directly from her profile](https://quitter.es/aleksandar) and it works perfectly. I've been testing a little more and realized that it happens only with notifications from users using quitter.es (the instance I use).

Should I open a new issue? if not, thanks again and see you on GS (and here)! :)
